core/src/main/java/io/aiven/klaw/controller/EnvsClustersTenantsController.java

```diff
@@ -195,6 +246,33 @@ public ResponseEntity<List<EnvModelResponse>> getKafkaConnectEnvs() {
         envsClustersTenantsControllerService.getKafkaConnectEnvs(), HttpStatus.OK);
   }
 
+  @RequestMapping(
+      value = "/environments/connect",
+      method = RequestMethod.GET,
+      produces = {MediaType.APPLICATION_JSON_VALUE})
+  public ResponseEntity<List<EnvModelResponse>> getKafkaConnectEnvsPaginated(
+      @RequestParam("pageNo") String pageNo,
+      @RequestParam(value = "searchEnvParam", defaultValue = "") String searchEnvParam) {
+    return new ResponseEntity<>(
+        envsClustersTenantsControllerService.getEnvsPaginated(
+            KafkaClustersType.KAFKA_CONNECT, "", pageNo, searchEnvParam),
+        HttpStatus.OK);
+  }
+
+  @RequestMapping(
+      value = "/environments/connect/{envId}",
```

Comment:
```suggestion
      value = "/environments/kafkaconnect/{envId}",
```
also here, better if we could be more specific